src/components/TokenIcon.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+import FastImage from 'react-native-fast-image'
```

Comment:
What's the benefit of this over just regular Image? Seems like this package hasn't been active for close to a year

Reply:
[FastImage](react-native-fast-image
) has some better caching as is more performant than the react native image component. Since we already use it in the NFTMedia component it doesn't seem like much of a risk to add here.